Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7-alpine
```

Comment:
Well, how are we planning to deploy the app?

Reply:
We gonna deploy in containers, but as I understand this configuration is for local development, since you copy instead of pulling from repository. Anyway we prefer first to develop and then to deploy. So the nudge is to focus on code right now.

Reply:
Firstly, flag `--dev` can be removed at any time. Thus, we can get a production ready image.

Secondly, you don't need to pull the repo to docker image.  Files from master branch is just copied into docker image. It's much faster, and support caching.

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

Reply:
Is it production-ready level when you have to have project codebase placed on the remote server fs?